integrationExamples/gpt/pbjs_example_gpt.html

```diff
@@ -104,6 +104,13 @@
                     }
             },
             {
+                bidder: 'nginad',
+                    params: {
+                        pzoneid: '7', // <String> PublisherAdZoneID
+                        nginad_domain: "server.nginad.com" // the domain where you installed NginAd, Omit leading http://

```

Comment:
Will this param return a bid or do we need to modify `nginad_domain`?
Also please use `camelCase` for variable names